File: hard/11_multi_string_search/multi_string_search_test.py
import unittest
from multi_string_search import multiStringSearch

class TestTraverseGraph(unittest.TestCase):
    def test_case_1(self):
        expected = [True, False, True, True, False, True, False]
        actual = multiStringSearch('this is a big string', ['this', 'yo', 'is', 'a', 'bigger', 'string', 'kappa'])
        self.assertEqual(actual, expected)

File: hard/11_multi_string_search/multi_string_search.test.js
const  multiStringSearch = require('./multi_string_search');

test("Test Case #1", () => {
  expect(multiStringSearch('this is a big string', ['this', 'yo', 'is', 'a', 'bigger', 'string', 'kappa'])).toEqual([true, false, true, true, false, true, false]);
});
